web/src/components/Header.js

```diff
@@ -1,22 +1,64 @@
-import React from 'react';
+import Button from '@cmsgov/design-system-core/dist/components/Button/Button';
+
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
 import { Link } from 'react-router-dom';
+import { push } from 'connected-react-router';
+
 
 import { t } from '../i18n';
 
-const Header = () => {
-  return (
-    <header className="header__slim">
-      <div className="ds-l-container">
-        <div className="ds-l-row">
-          <div className="ds-l-col">
-            <Link to="/" className="site-title">
-              {t('titleBasic')}
-            </Link>
+class Header extends Component {
+  handleLogout = e => {
+    e.preventDefault();
+    const { pushRoute } = this.props;
+    pushRoute('/logout');
+  };
```

Comment:
I stole this and the other auth checks from `TopBtns.js`. Since this is half-baked still, I thought it wasn't worth adapting the tests (but I can if @mgwalker disagrees with me).